pom.xml

```diff
@@ -61,18 +70,6 @@
     <build>
         <pluginManagement>
             <plugins>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-javadoc-plugin</artifactId>
-                    <version>3.5.0</version>
-                </plugin>
-                <plugin>
-                    <groupId>org.sonatype.plugins</groupId>
-                    <artifactId>nexus-staging-maven-plugin</artifactId>
-                    <configuration>
-                        <autoReleaseAfterClose>true</autoReleaseAfterClose>
-                    </configuration>
-                </plugin>
```

Comment:
That's part of the updated parent pom

Reply:
Could probably also upstream https://github.com/jenkinsci/archetypes/blob/00a88f9ecec7cca427f3500e3c1055586d7bc894/pom.xml#L83 & https://github.com/jenkinsci/archetypes/blob/00a88f9ecec7cca427f3500e3c1055586d7bc894/pom.xml#L88-L92 (do not recall original reasons for these overrides) and https://github.com/jenkinsci/archetypes/blob/00a88f9ecec7cca427f3500e3c1055586d7bc894/pom.xml#L67

Reply:
Good idea, I've moved them in 1.1